docs/4.2/admin-guide.md

```diff
@@ -2038,8 +2038,7 @@ To make this work, the Teleport proxy server must be able to access a Kubernetes
 API endpoint. This can be done either by:
 
 * Deploying the proxy service inside a Kubernetes pod.
-* Deploying the proxy service outside Kubernetes adding a valid `kubeconfig`
-  setting to the configuration file as shown above.
+* Deploying the proxy service outside Kubernetes adding a valid `kubeconfig` setting to the   configuration file as shown above.
```

Comment:
We should add a comma after Kubernetes (`Kubernetes, adding`)
Also random triple space in `the    configuration file`